build.gradle

```diff
@@ -81,6 +81,10 @@ if (JavaVersion.current().isJava11Compatible()) {
           // We import it as api dependency via :servicetalk-annotations
           exclude("com.google.code.findbugs:jsr305")
         }
+        onCompileOnly {
+          // For backward compatibility
+          exclude(":servicetalk-annotations") // FIXME: 0.43 - move to compileOnly
```

Comment:
How do you guys think, can we move it to `compileOnly` scope now or should we keep it as `implementation` for 0.42.x branch?

Reply:
I'm not 100% sure on the impact, could it break downstream users who rely on servicetalk-annotations be on the classpath somehow now to include it manually? If so, I would keep it in 0.42.x as implementation to be safe.

Reply:
Hm, I tried and got many build failures. It looks like a false positive because this annotation is marked with `RetentionPolicy.RUNTIME` and not `SOURCE`. So, we should keep it as-is

Reply:
Do you plan on giving me an approval? 😄

Reply:
yes sir 🫡